src/components/LoginForm/LoginForm.js

```diff
@@ -1,12 +1,17 @@
 import React, { Component } from 'react';
+import { Redirect, Link } from 'react-router-dom';
 import { PropTypes } from 'prop-types';
 import { connect } from 'react-redux';
 import * as actions from '../../actions';
+import requests from '../../utils/requests/requests';
 
 export class LoginForm extends Component {
   state = {
     username: '',
-    password: ''
+    password: '',
+    success: false,
+    error: '',
+    loading: false
```

Comment:
We also have a isFetching state in Redux, do we want to utilize that instead of local state?

Reply:
I think it would be more efficient if its local state since the form itself is the only component that needs to know if the login was good to go.
When we start using thunks, that would be a good use-case for the redux state.